packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.constant.js

```diff
@@ -5,6 +5,19 @@ export const USER_TYPE = {
   default: 'others',
 };
 
+export const LEGAL_LINK1 =
+  'https://www.cert-in.org.in/PDF/CERT-In_Directions_70B_28.04.2022.pdf';
+export const LEGAL_LINK2 = 'https://www.cert-in.org.in/Directions70B.jsp';
+export const LEGAL_LINK3 = {
+  FR: 'https://www.ovhcloud.com/fr/terms-and-conditions/privacy-policy/',
+  DE: 'https://www.ovhcloud.com/de/terms-and-conditions/privacy-policy/',
+  ES: 'https://www.ovhcloud.com/es-es/terms-and-conditions/privacy-policy/',
+  IT: 'https://www.ovhcloud.com/it/terms-and-conditions/privacy-policy/',
+  PL: 'https://www.ovhcloud.com/pl/terms-and-conditions/privacy-policy/',
+  PT: 'https://www.ovhcloud.com/pt/terms-and-conditions/privacy-policy/',
+  OTHERS: 'https://www.ovhcloud.com/en-gb/terms-and-conditions/privacy-policy/',
```

Comment:
Keep the default name.
We use DEFAULT for all others subs.
```suggestion
  DEFAULT: 'https://www.ovhcloud.com/en-gb/terms-and-conditions/privacy-policy/',
```